frontend/scripts/pr_deploy.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    echo "Not a PR. Skipping surge deployment"
+    exit 0
+fi
+
+ember build production
+
+npm i -g surge
+
+export SURGE_LOGIN=test@example.co.in
+# Token of a dummy account. So can be added to vcs
+export SURGE_TOKEN=d1c28a7a75967cc2b4c852cca0d12206
+
+export DEPLOY_DOMAIN=https://pr-${TRAVIS_PULL_REQUEST}-fossasia-badgeyay.surge.sh
+surge --project ./dist --domain $DEPLOY_DOMAIN;
```

Comment:
Dont we need to rename index.html to 200.html in dist directory before deploying ? Its written in the docs of surge, that's why asking. 
Otherwise LGTM 👍

Reply:
I think ember build takes care of that @rupav